base_user_role/wizards/group_groups_into_role.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Sodexis
+# License OPL-1 (See LICENSE file for full copyright and licensing details).
+
+
+from odoo import fields, models
+
+
+class GroupGroupsIntoRole(models.TransientModel):
+    """
+    This wizard is used to group different groups into a role.
+    """
+
+    _name = "group.groups.into.role"
```

Comment:
```suggestion
    _name = "wizard.groups.into.role"
```

I think it's great if the name of the model is explicit and mention that it is transient. I think it's a norm in the odoo / oca modules, don't you think ?